### ISSUE TITLE
Comparing general layout of My Events and Event Group pages. Adding an adjustment to the Event Group page.

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#133]
I am comparing the general layout of 
**Manage Events**

My Events:
![Screen Shot 2021-07-25 at 16 11 50](https://user-images.githubusercontent.com/5323259/126902184-05b29c8d-be96-45e5-812d-bb20445c685c.png)
 
and

**Manage Event Groups**

Event Groups
![Screen Shot 2021-07-25 at 16 14 32](https://user-images.githubusercontent.com/5323259/126902261-cd3f5ae7-ae5b-4b49-a5c0-debbf2e32e04.jpg)

A suggestion for adjusting the Manage Event Groups

![Manage-Event-Groups-adjustment-ver1](https://user-images.githubusercontent.com/5323259/126902589-a53172fe-025c-4631-a757-95110901d479.jpg)

Reasons why I adjusted the Manage Events Groups page:

This information is not needed at this stage:
"When you choose a group of events to edit, the form will be pre-filled with the content from the event you started from. You will also see a set of checkboxes to choose which events you want to apply these changes to."
(I just find it confusing having it there, as I need to enter a grouped event and look through to see what I need to do.)

Having a section with text:
Manage Event Groups 
- Select an event group to edit.
(Not needed. As the user is here to edit something.)

Having the text: "Check a set of events to group them for mass editing." is almost the same as the text in the blue button.

I made the screen look a lot more similar to the My Events page. Creating a kind of visual consistency between the screens.

**Comment by paaljoachim** [comment]
Another thing...
I noticed the bottom button is grey and on the right side. The top button is blue and on the left.

![Screen Shot 2021-07-25 at 16 34 02](https://user-images.githubusercontent.com/5323259/126902932-0fd2e75a-91be-44f8-900a-0e17c6d5991c.png)

I will assume that the button in this screen should look similar to the My Events buttons. 
Be greyed out when not active. (grey button and text.)
When clicking the event checkbox having the grey text become blue, along with a blue border.


https://user-images.githubusercontent.com/5323259/126903081-a0c80570-a3ca-4f78-9304-c81f3fb9cd77.mp4

**Comment by joedolson** [comment]
I think that I've done everything here in those four commits. Closing!

**Comment by paaljoachim** [comment]
Great!

![Screen Shot 2021-07-26 at 00 06 48](https://user-images.githubusercontent.com/5323259/126914914-a9481d44-a19a-43b0-839b-478ca4bbd8f0.png)

I also like the bigger tabs and size 14px fonts.

This text: (Is it needed?)
"When you choose a group of events to edit, the form will be pre-filled with the content from the event you started from. You will also see a set of checkboxes to choose which events you want to apply these changes to."

**Comment by joedolson** [comment]
Honestly, you're probably better equipped to answer that question. If you arrived to this page with no instructions, would you know what to do or how it worked?

**Comment by paaljoachim** [comment]
Thinking out loud...
I create an event that has multiple occurrences. 
I go to the Manage Event Groups page. Read "When you choose a group of events to edit, the form will be pre-filled with the content from the event you started from." 
I would expect this as the occurrences would use the same information except date and time.

I then read: "You will also see a set of checkboxes to choose which events you want to apply these changes to."
Now that confuses. As it talks about something I will see when clicking to edit and event group.

After clicking the event group there are better instructions there than the initial event group instructions.
![Screen Shot 2021-07-26 at 00 24 57](https://user-images.githubusercontent.com/5323259/126915293-e4ff3f5d-c57c-4318-bf14-0160b79ae072.png)

(I would rephrase these instructions. I will look into it in a moment.)

**Comment by joedolson** [comment]
The fact that the initial data in the group editing view is the data for the specific event you choose does matter. If the data *doesn't* match, that controls which data will be propagated through all of the events you edit. 

E.g., if you take two existing events that have different descriptions, then group them, you can edit them so that they'll all have the same description. You'd pick the event with the description you want to keep to do that. 

I find that a difficult thing to convey. 

Agree that telling people in advance that "there will be checkboxes to check" is irrelevant. They'll be obvious when they get there...

**Comment by paaljoachim** [comment]
Heck.... this is pretty complex. I have yet to merge two different single events. 
Looking into it now....
